src/frontend/org/voltdb/HTTPClientInterface.java

```
@@ -164,6 +164,27 @@ public void process(Request request, HttpServletResponse response) {
 
             String procName = request.getParameter("Procedure");
             String params = request.getParameter("Parameters");
+            String timeoutStr = request.getParameter("ProcTimeout");
+            int procCallTimeout = -1;
+            if (timeoutStr != null) {
+                try {
+                    procCallTimeout = Integer.parseInt(timeoutStr);
+                    if (procCallTimeout <= 0) {
+                        throw new NumberFormatException();
```

They passed in a bad argument value. I think its better to abort than silently use a default value.